src/controllers/auth.js

```diff
@@ -14,20 +18,31 @@ class AuthController {
         password,
       } = req.body;
         // Check if the email already exists
-      const user = await UserModel.findOne({ where: { email, } });
+      const user = await User.findOne({
+        where: {
+          email
+        }
+      });
       if (user) {
-        res.status(400).json({ status: 400, error: 'Email already exists', });
+        res.status(400).json({
+          status: 400,
+          error: 'Email already exists'
+        });
       } else {
         // Password hash occurs in the hook beforeSave
-        const newUser = await UserModel.create({
+        const newUser = await User.create({
           firstName,
           lastName,
           email,
           password,
         });
           // Generate JWT
-        const token = jwt.sign({ id: newUser.id, email, }, process.env.JWT_SECRET, { expiresIn: '50h' });
-        return res.status(201).json({
+        const token = jwt.sign({ id: newUser.id, email },
```

Comment:
Since you don't primarily have to work in this controller, i think you shouldn't have changed things here, so i can see merge conflicts prepared for me already lol .